### DISCUSSION TITLE
How do I use `gh pr merge` to set a branch to auto-merge?

### DISCUSSION THREAD

**Discussion by Industrial** [community/community#81590] [Pull Requests]
### Select Topic Area

Question

### Body

Hello.

I want to use the `auto-merge` feature on pull requests. I have checked the boxes and enable the feature on my repository so it should work.

When I run the commands (as part of a script):
```bash
gh pr create --title "${BRANCH_NAME}"
gh pr merge --auto --delete-branch --squash "${BRANCH_NAME}"
```
Then what that does is:
1. Ask me some questions about creating the PR.
1. Create the PR.
1. When the actions are still running: I get this error`GraphQL: Pull request Pull request is in unstable status ("enablePullRequestAutoMerge")`
1. When the actions are finished: Immediately merges.

What I expected it to do was to mark the PR as auto-mergable and leave the merging for when the actions (and other checks) pass and immediately quit the program.

How do I achieve this?

**Comment by queenofcorgis** [comment]
Hi there @Industrial - thanks for posting this!

Are you using this on a Free plan private repository by chance?
